### PR TITLE
chore(demo): add new examples to documentation page `Animations`

### PR DESCRIPTION
**Pull request by VlaskK**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [X] Documentation content changes

Partially solves https://github.com/taiga-family/taiga-ui/issues/684

### CODE REVIEW

File: package-lock.json
Comment (n-barsukov) [inline package-lock.json:1]: Revert this file

File: projects/demo/src/modules/animations/examples/dropdown/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<button
+    appearance="outline"
+    tuiButton
+    type="button"
+    class="button"
```
Comment (n-barsukov) [inline projects/demo/src/modules/animations/examples/dropdown/index.html:5]: Do we need this class for any customization ?

File: projects/demo/src/modules/animations/examples/dropdown/index.less [hunk 2]
```diff
@@ -0,0 +1,9 @@
+.dropdown {
+    position: absolute;
+    background-color: #fff;
+    border: 1px solid gray;
```
Comment (n-barsukov) [inline projects/demo/src/modules/animations/examples/dropdown/index.less:4]: Use colors from palette
dark-theme

File: projects/demo/src/modules/animations/examples/dropdown/index.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    encapsulation,
+    animations: [tuiDropdownAnimation],
+})
+export class TuiDropdownExample {
+    @Input()
+    speed = 0;
+
+    isOpen = false;
+
+    items = ['Angular', 'React', 'Vue'];
```
Comment (nsbarsukov) [inline projects/demo/src/modules/animations/examples/dropdown/index.ts:21]: Do it used anywhere ?

File: projects/demo/src/modules/animations/examples/slide-in/index.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+    animations: [
+        tuiSlideInLeft,
+        tuiSlideInTop,
+        tuiSlideInBottom,
+        tuiSlideInRight,
+        tuiSlideInLeftList,
```
Comment (nsbarsukov) [inline projects/demo/src/modules/animations/examples/slide-in/index.ts:25]: Do we need it for our example ?

**Review by splincode — APPROVED** [review]

File: projects/core/styles/mixins/mixins.scss
Comment (nsbarsukov) [inline projects/core/styles/mixins/mixins.scss:1]: Revert this file, please

File: projects/demo/src/styles.scss
Comment (nsbarsukov) [inline projects/demo/src/styles.scss:1]: Revert this file, please

**Comment by splincode** [comment]
@VlaskK hello, please add changes from main branch

File: projects/demo/src/modules/animations/examples/slide-in/index.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    reset(): void {
+        this.isLeft = false;
+
+        this.isTop = false;
+
+        this.isRight = false;
+
+        this.isBottom = false;
```
Comment (splincode) [inline projects/demo/src/modules/animations/examples/slide-in/index.ts:45]:
```ts
this.isLeft = false;
      this.isTop = false;
      this.isRight = false;
      this.isBottom = false;
```

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**